File: src/main/scala/com/bombproj/model/ProjectDatabase.java
package com.bombproj.model;

import com.queryflow.annotation.Column;
import com.queryflow.annotation.Table;
import lombok.Data;

import java.util.Date;

@Data
@Table("project_database")
public class ProjectDatabase {

    @Column(exists = false)
    private Long id;

    private String databaseName;
    private String databaseDesc;
    private String type;
    private String dbHost;
    private String dbPort;
    private String userName;

    private String testDbHost;
    private String testDbPort;
    private String testUserName;

    private String prodDbHost;
    private String prodDbPort;
    private String prodUserName;

    private String version;
    private Integer state;
    private String projectId;
    private String userId;
    private Date createTime;
    private Date updateTime;

}

File: src/main/scala/com/bombproj/model/ProjectDatabase.scala
package com.bombproj.model

import java.util.Date

import com.queryflow.annotation.{Column, Table}

import scala.beans.BeanProperty

@Table("project_database")
case class ProjectDatabase() {

    @BeanProperty
    @Column(exists = false) var id: Long = _

    @BeanProperty var databaseName: String = _
    @BeanProperty var databaseDesc: String = _
    @BeanProperty var `type`: String = _
    @BeanProperty var dbHost: String = _
    @BeanProperty var dbPort: String = _
    @BeanProperty var userName: String = _
    @BeanProperty var testDbHost: String = _
    @BeanProperty var testDbPort: String = _
    @BeanProperty var testUserName: String = _
    @BeanProperty var prodDbHost: String = _
    @BeanProperty var prodDbPort: String = _
    @BeanProperty var prodUserName: String = _
    @BeanProperty var version: String = _
    @BeanProperty var state: Int = _
    @BeanProperty var projectId: String = _
    @BeanProperty var userId: String = _
    @BeanProperty var createTime: Date = _
    @BeanProperty var updateTime: Date = _

}
